### PR TITLE
[PERFSCALE-3608] BM CPT: Foreman deploys

### PR DESCRIPTION
**Pull request by josecastillolema**

### CODE REVIEW

**Comment by josecastillolema** [comment]
@radez

File: ci-operator/step-registry/openshift-qe/installer/bm/foreman/openshift-qe-installer-bm-foreman-commands.sh [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+set -x
+
+# Fix UID issue (from Telco QE Team)
+~/fix_uid.sh
+
+bastion=$(cat "/secret/address")
+
+# Set the corresponding host on the lab Foreman instance
+sshpass -p "$(cat /secret/login)" ssh -oStrictHostKeyChecking=no -oUserKnownHostsFile=/dev/null root@${bastion} "
+  NUM_NODES_CLOUD=$(curl -sS http://$QUADS_INSTANCE/cloud/$LAB_CLOUD\_ocpinventory.json | grep name | wc -l)
```
Comment (jtaleric) [inline ci-operator/step-registry/openshift-qe/installer/bm/foreman/openshift-qe-installer-bm-foreman-commands.sh:14]: In rdu2 doesn't quads use httpS not http?

Comment (jtaleric) [reply ci-operator/step-registry/openshift-qe/installer/bm/foreman/openshift-qe-installer-bm-foreman-commands.sh:14]: might be easier to have the user provide the URL to quads for us to append to?

**Review by jtaleric — APPROVED** [review]
lgtm

**Comment by josecastillolema** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-metal-x86-foreman-1node

**Comment by josecastillolema** [comment]
https://github.com/openshift/release/blob/master/ci-operator/step-registry/openshift-qe/node-density-heavy/openshift-qe-node-density-heavy-commands.sh#L62-L65

**Comment by josecastillolema** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-metal-x86-foreman-1node

**Comment by josecastillolema** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-metal-x86-foreman-1node

**Comment by josecastillolema** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-metal-x86-foreman-1node

**Comment by rsevilla87** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-metal-x86-foreman-1node

**Pull request closed without merging**